src/ui/public/vis/vis.js

```diff
@@ -75,9 +84,21 @@ export function VisProvider(Private, Promise, indexPatterns, timefilter, getAppS
         queryFilter: queryFilter,
         queryManager: queryManagerFactory(getAppState),
         events: {
-          filter: (event) => {
+          filter_legacy: (event) => {
```

Comment:
Could we please add a comment to this, why it's legacy, and that it should not be used anymore, because we will remove it at some point.